substrate/utils/frame/benchmarking-cli/src/pallet/command.rs

```diff
@@ -298,16 +298,22 @@ impl PalletCmd {
 		// Convert `Vec<u8>` to `String` for better readability.
 		let benchmarks_to_run: Vec<_> = benchmarks_to_run
 			.into_iter()
-			.map(|b| {
+			.map(|(pallet, extrinsic, components, pov_modes)| {
+				let pallet_str = String::from_utf8(pallet.clone()).expect("Encoded from String; qed");
+				let extrinsic_str = String::from_utf8(extrinsic.clone()).expect("Encoded from String; qed");
+				let pov_modes_str = pov_modes.into_iter()
```

Comment:
**Nitpick**. In general we don't use [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) or any variation of it. So in this case I'd rename these variables to just `pallet_name`, `extrinsic_name` and `pov_modes` (as done some files below - writer.rs?)

Reply:
Yep, sure thing. I've updated the variable names for `pallet_str`, `extrinsic_str` and removed `pov_modes_str` as `pov_modes` was already being passed in